src/models/parties/parties.js

```diff
@@ -6,40 +6,42 @@ export const DEFAULT_PARTY_COUNT = 5;
 export const MAX_PARTY_COUNT = 10;
 
 export function makeParties() {
-  let names = Set([
-    'asparagus',
-    'corn',
-    'cabbage',
-    'turnip',
-    'radish',
-    'carrot',
-    'onion',
-    'spinach',
-    'mushroom',
-    'eggplant'
-  ]);
+  let names = [
```

Comment:
Since this is no longer a list only names, `names` is not an appropriate variable name. A better option might be `potentialParties`.